src/connectedmachine/azext_connectedmachine/aaz/latest/connectedmachine/extension/_create.py

```diff
@@ -91,12 +91,12 @@ def _build_arguments_schema(cls, *args, **kwargs):
 
         _args_schema = cls._args_schema
         _args_schema.auto_upgrade_minor_version = AAZBoolArg(
-            options=["--upgrade-minor-ver"],
+            options=["--auto-upgrade-minor", "--auto-upgrade-minor-version"],
             arg_group="Properties",
             help="Indicates whether the extension should use a newer minor version if one is available at deployment time. Once deployed, however, the extension will not upgrade minor versions unless redeployed, even with this property set to true.",
         )
         _args_schema.enable_automatic_upgrade = AAZBoolArg(
-            options=["--enable-auto-upgrade"],
+            options=["--auto-upgrade", "--enable-automatic-upgrade"],
```

Comment:
Let me check with you if this param is renamed and the current name `--enable-auto-upgrade` is deprecated? If so, it is better to add the current name back to aviod breaking change.
```suggestion
            options=["--enable-auto-upgrade", "--auto-upgrade", "--enable-automatic-upgrade"],
```

Reply:
updated